testing/src/test/java/io/stargate/it/CQLTest.java

```diff
@@ -88,8 +87,8 @@ public void setup() {
     table = testName;
   }
 
-  @BeforeClass
```

Comment:
We shouldn't be using static state here because it relies on non-static state `stargateHost` in `BaseOsgiIntegrationTest`. If a test is not run beforehand then `stargateHost` will be `null`.